tests/test_bundle.py

```diff
@@ -172,7 +172,7 @@ def _test_bundle_get_paging(self,
                 # Make sure we're getting the expected response status code
                 if link_header:
                     self.assertEqual(resp_obj.response.headers['X-OpenAPI-Pagination'], 'true')
-                    self.assertEqual(resp_obj.response.headers['X-OpenAPI-Paginated-Content-Key'], 'files')
+                    self.assertEqual(resp_obj.response.headers['X-OpenAPI-Paginated-Content-Key'], 'bundle.files')
```

Comment:
This LGTM, though in every case the `X-OpenAPI-Paginated-Content-Key` should be present and equal to `bundle.files`, so it could make sense to move L175 outside of the `if link_header` block.

Reply:
addressed